docker/Dockerfile.xpdf

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:latest
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    build-essential \
+    cmake \
+    curl \
+    gcc \
+    git \
+    libtesseract-dev \
+    poppler-utils \
+    tesseract-ocr
+
+ARG xpdf_version
+RUN curl -O https://dl.xpdfreader.com/xpdf-${xpdf_version}.tar.gz && \
+    tar -xvf xpdf-${xpdf_version}.tar.gz && \
+    cd xpdf-${xpdf_version} && \
+    cmake . && \
+    make && \
+    cp xpdf/pdftotext /opt && \
+    rm -rf ../xpdf-${xpdf_version}
```

Comment:
TIL removing recursively the CWD works like a charm 😅

Reply:
Good point, changing that. 😅